test/kuttl/tests/change_neutron_config/02-assert.yaml

```diff
@@ -0,0 +1,5 @@
+# check that by default, debug is set in neutron config
```

Comment:
why this file has name "02-assert" and not "01-assert"?

Reply:
we have already 01-assert file, that uses the assert_sample_deployment. 02-assert checking only the debug config (without 02-action_file)